requirements.txt

```diff
@@ -5,3 +5,9 @@ dwave-preprocessing==0.3.0
 reno==3.3.0  # for changelog
 
 setuptools>=46.4.0  # to support setup.cfg getting __version__
+
+dimod~=0.10.5
```

Comment:
Only `pyparsing` should be in here. The others are in `tests/requirements.txt`.

Reply:
whoops.. I let pycharm do its job... badly